server/datastore/mysql/apple_mdm.go

```diff
@@ -189,6 +189,19 @@ WHERE
 		return nil, ctxerr.Wrap(ctx, err, "get mdm apple config profile")
 	}
 
+	// get the labels for that profile, except if the profile was loaded by the
+	// old (deprecated) endpoint.
+	if uuid != "" {
+		labels, err := ds.listProfileLabelsForProfiles(ctx, nil, []string{res.ProfileUUID})
```

Comment:
I _think_ we want to add the `labels` to the Get profile endpoint too (not just the list profiles one). I asked here to confirm: https://github.com/fleetdm/fleet/pull/15117/files#r1465289567 but it makes sense to me.